typescript-selenium/utils/NameGenerator.ts

```diff
@@ -0,0 +1,26 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+export class NameGenerator {
+    public static generate(prefix: string, randomLength: number): string {
+        let possibleCharacters: string = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
```

Comment:
Dashboard uses single quotes for strings, you should probably unify. Again, should be checked by tslint.